config.json

```diff
@@ -15,6 +15,20 @@
     "indent_size": 4
   },
   "test_pattern": ".*[.]test[.]ts$",
+  "files": {
+    "solution": [
+      "%{kebab_slug}.ts"
+    ],
+    "test": [
+      "%{kebab_slug}.test.ts"
+    ],
+    "example": [
+      "%{kebab_slug}.example.ts"
```

Comment:
I know the JS track has renamed these files to something with `proof` in it. My suggestion would be to get this merged first and fix it later.